wger/exercises/api/views.py

```diff
@@ -46,6 +48,29 @@
 from wger.utils.permissions import CreateOnlyPermission
 
 
+class ExercisesListSpecial(APIView):
+    @classmethod
```

Comment:
@Manorlds-Eaglespark are you accessing anything on the class in this method? Otherwise, it's not necessary to make it a `class method`. A simple override of the `get` method on the `APIView` would suffice.